### PR TITLE
fix(TextArea/native): modify label and accessory position

### PR DESCRIPTION
**Pull request by deorukhkarchaitanya**

### CODE REVIEW

File: src/atoms/TextInput/AccessoryIcon.native.js [hunk 1]
```diff
@@ -42,12 +46,14 @@ AccessoryIcon.propTypes = {
   disabled: PropTypes.bool,
   hasError: PropTypes.bool,
   variant: PropTypes.oneOf(['filled', 'outlined']).isRequired,
+  _isMultiline: PropTypes.bool,
```
Comment (kamleshchandnani) [inline src/atoms/TextInput/AccessoryIcon.native.js]: This is basically mixing concerns. AccessoryIcon, AccessoryText should not be aware whether they are part of multiline or not.

Comment (kamleshchandnani) [reply src/atoms/TextInput/AccessoryIcon.native.js]: Everything is not flowing here and there. Maybe we can rethink this. Let me give it more thougts

Comment (deorukhkarchaitanya) [reply src/atoms/TextInput/AccessoryIcon.native.js]: TextInput & TextInput with multiline prop has very different visual properties on Android & iOS. Right now these accessories were not aligning with TextInput multiline but are aligning with TextInput.

Comment (deorukhkarchaitanya) [reply src/atoms/TextInput/AccessoryIcon.native.js]: The UI is broken without this. I suggest we merge this for now & revisit it later. No point keeping it half baked

**Pull request merged**